### DISCUSSION TITLE
Canadian Federal Tax Numbers

### DISCUSSION THREAD

**Discussion by oldskeptic** [edmcouncil/fibo#1501] [Business Entities (BE)] [answered]
I am working on a project where I need to store the Canadian Federal Tax Business Numbers and the Canadian Federal Tax Accounts for Corporate Tax (RC), Payroll (RP), HST (TR), etc... for business entities.

Currently FBO has the [fibo-be-le-fbo:ValueAddedTaxIdentificationNumber](https://github.com/edmcouncil/fibo/blob/89248dfec55fce57fd6cbbf4adf5a0cb20148b16/BE/LegalEntities/FormalBusinessOrganizations.rdf#L217), [fibo-be-le-fbo:TaxIdentifier](https://github.com/edmcouncil/fibo/blob/89248dfec55fce57fd6cbbf4adf5a0cb20148b16/BE/LegalEntities/FormalBusinessOrganizations.rdf#L295) and [fibo-be-le-fbo:OrganizationIdentifier](https://github.com/edmcouncil/fibo/blob/89248dfec55fce57fd6cbbf4adf5a0cb20148b16/BE/LegalEntities/FormalBusinessOrganizations.rdf#L165) classes that can be subclasses for the Canadian Federal Jurisdiction context. What is the process for getting new terms approved before a pull request is sent?

**Comment by ElisaKendall** [comment, marked as answer]
@oldskeptic Hi -  your extensions should be in FBC, in FunctionalEntities, in a new CARegulatoryAgencies ontology under NorthAmericanEntities ontology.  You might look at the USRegulatoryAgencies ontology for some quidance.  We have specified EmployerIdentificationNumber, for example, in that ontology that you could look at for guidance.  We also specify Account in FBC, so any account specific details would necessarily go in FBC in any case.  

Next steps would be to join us on our telecon next week, Monday, May 3rd, at 7:30 am pacific / 10:30 am eastern.  Dial in details are in our wiki at https://wiki.edmcouncil.org/display/SEC/.  If you don't have an account on the wiki, I think you can create one, so please try that, and if it doesn't work, please send me email directly, at ekendall@thematix.com.

**Reply by oldskeptic** [reply]
Sounds grand. I will join the conference and leave this discussion open for notes for the time being.

No joy on the account. I will email you directly.
